### ISSUE TITLE
Create basic Angular project

### ISSUE THREAD

**Issue by AlexanderMandera** [AlexanderMandera/Transport-Desktop#2]
Create a basic Angular project with the Angular CLI and implement Router, etc.

**Comment by LKaemmerling** [comment]
Just as an idea:

Use Ionic (v4 https://ionicframework.com/) then you could use the same code base for possible more platforms (like Android, iOS and co). It is still Angular (6)

**Comment by sinuscosinustan** [comment]
You know, that this App will be only developed for Desktop (Windows, Mac, Linux).

**Comment by MarcelReif** [comment]
We'll first release Transport as a desktop application for Windows, macOS and Linux. As far as the beta phase is done we'll maybe also start developing apps for Android and iOS. Please stay tuned.

**Comment by LKaemmerling** [comment]
Yeah, but many apps that were first developed for Desktop only make the switch to mobile too. Ionic is not only for mobile. With v4 it contains now: https://stenciljs.com/ and https://capacitor.ionicframework.com/ and is great for desktop development too! And of course, the way for mobile is open too.
